src/ssh/cockpitsshrelay.c

```diff
@@ -2094,6 +2098,13 @@ cockpit_ssh_relay_start (CockpitSshRelay *self,
     .channel_exit_status_function = on_channel_exit_status,
   };
 
+  lang = g_getenv ("LANG");
+  lc_msg = g_getenv ("LC_MESSAGES");
+  command = g_strdup_printf ("LANG=%s LC_MESSAGES=%s %s",
+                             lang ? lang : "C",
+                             lc_msg ? lc_msg : "C",
```

Comment:
This is wrong, I think - if you only set `$LANG` and not `$LC_MESSAGES`, then it shouldn't be passed at all, or set to the same value as `LANG` (whichever is easier, should have the same net effect). Otherwise `LC_MESSAGES=C` will override `LANG=<whatever`>`. So rather something like

```c
   command = g_strdup_printf ("LANG=%s %s%s %s",
                             lang ?: "C",
                             lc_msg ? "LC_MESSAGES=" : ""
                             lc_msg ?: "",
```

or, if you like that better, setting it to the same value as lang:

```c
                             lc_msg ?: (lang ?: "C"),
```

(Also, I wonder if at some point we should default to C.utf8 to avoid breaking fancy symbols from program outputs. But this is unrelated to this PR.)

Reply:
We prefer LC_MESSAGES to LANG, both are set by cockpit-ws to the same value, so generally it's always going to be the same. I'll change it to set LC_MESSAGES to lang if it isn't already set.

Reply:
Oh right, so I guess it doesn't make much of a difference then.